domovoi.py

```diff
@@ -58,7 +58,9 @@ def run(self, processes_file):
         self.check_paths(solar_car_processes)
         self.start_processes(solar_car_processes)
         # Watch over the processes and respond accordingly
-        while len(solar_car_processes): # Domovoi continues to run as long as there are processes in the list, other wise it quits
+        while True:
+            if len(solar_car_processes) == 0:
```

Comment:
Do the check before the while loop. Reduce nesting where possible :P I know these things seem nitpicky, but generally python looking pythonic has a real look to it that is meaningful.